bin/config.toml

```diff
@@ -1,7 +1,7 @@
 # sozu proxy test config file
 
-command_socket = "./command_folder/sock"
-saved_state    = "./state.json"
+command_socket = "/var/run/sozu/socket"
```

Comment:
please do not modify this test file, make a separate template config for packaging